rasterio/rio/edit.py

```diff
@@ -0,0 +1,65 @@
+"""Edit raster dataset metadata from the command line."""
+
+import json
+import logging
+
+import click
+
+import rasterio
+from rasterio.rio.cli import cli, file_in_arg
+from rasterio.transform import guard_transform
+
+
+@cli.command(short_help="Edit dataset metadata.")
+@file_in_arg
+@click.option('--nodata', type=float, default=None,
+              help="New nodata value")
```

Comment:
@sgillies I'm trying to understand your choice of single vs double quotes for strings.  Is there a particular pattern you are going for here?